angr/analyses/cfg_accurate.py

```diff
@@ -990,7 +990,8 @@ def _process_hints(self, analyzed_addrs, remaining_entries):
                                           new_state,
                                           self._context_sensitivity_level
                                           )
-                remaining_entries.append(new_path_wrapper)
+                self._insert_entry(new_path_wrapper)
```

Comment:
The problem is the `remaining_entries` that gets passed in is actually the `_pending_entries` dict. So the append will fail, since that's a list method being called on a dict instance. 

Second, the `new_path_wrapper` is a `CFGJob` and when `_clean_pending_exits` is called it assumes all items in `_pending_entries` are of the class `PendingExit`.